scripts/add_export.py

```diff
@@ -219,6 +208,7 @@ def create_export_profile():
             discountrate="0.071",
             demand="AB",
             h2export="120",
+            # configfile="test/config.test1.yaml",
```

Comment:
Is this comment needed?

Reply:
We can drop this but it is handy when testing tutorial or test configurations